src/pages/careers/_join_your_team.tsx

```diff
@@ -195,6 +195,14 @@ const JoinYourTeam = () => {
             tagline:
                 'We take care of collaboration details to ensure that projects are completed in a timely, efficient manner.',
         },
+        {
+            img_data: data.projectManagement,
```

Comment:
Let's have a separate image for BI team, if not available in design need to get from UX team

Reply:
sure @habib-deriv i will check with them.